Terminal.Gui/Configuration/ScopeJsonConverter.cs

```diff
@@ -1,140 +1,189 @@
-﻿using System;
-using System.Linq;
+﻿#nullable enable
+using System.Diagnostics;
```

Comment:
Since this one has appeared in a few places, as well, this should probably be a global using, as well.